admin_guide/managing_projects.adoc

```diff
@@ -116,18 +116,18 @@ Subjects:
   Group	system:authenticated:oauth
 ----
 
-. Remove the `self-provisioners`
+. Remove the `self-provisioner`
 xref:../architecture/additional_concepts/authorization.adoc#roles[cluster role binding]
 from the group `system:authenticated:oauth`.
```

Comment:
There is the only confused point, should changed to `3.Remove the self-provisioner cluster role from the group system:authenticated:oauth.`

Reply:
Got it. Thank you @stuartchuan!